content/intro-to-storybook/angular/es/composite-component.md

```diff
@@ -224,50 +227,52 @@ A medida que el componente crece, también lo hacen los parámetros de entrada r
 
 ## Pruebas automatizadas
 
-En el capítulo anterior aprendimos a capturar historias de prueba utilizando Storyshots. Con el componente `TaskComponent` no había mucha complejidad para probar más allá de que se renderice correctamente. Dado que `TaskListComponent` añade otra capa de complejidad, queremos verificar que ciertas entradas produzcan ciertas salidas de una manera adecuada con pruebas automáticas. Para hacer esto crearemos test unitarios utilizando [Jest](https://facebook.github.io/jest/) junto con un renderizador de prueba.
+En el capítulo anterior aprendimos a capturar historias de prueba utilizando Storyshots. Con el componente `TaskComponent` no había mucha complejidad para probar más allá de que se renderice correctamente. Dado que `TaskListComponent` añade otra capa de complejidad, queremos verificar que ciertas entradas produzcan ciertas salidas de una manera adecuada con pruebas automáticas. Para hacer esto crearemos test unitarios utilizando [Angular Testing Library](https://testing-library.com/docs/angular-testing-library/intro).
 
 ![Jest logo](/intro-to-storybook/logo-jest.png)
 
-### Test unitarios con Jest
+<div class="aside">
+TODO:ADD testing library image
+</div>
```

Comment:
@josegarciamanez can you check this. It seems that this div should not be present.